plugins/sandbox/controller.go

```diff
@@ -83,6 +83,25 @@ type controllerLocal struct {
 
 var _ sandbox.Controller = (*controllerLocal)(nil)
 
+func (c *controllerLocal) cleanupShim(ctx context.Context, sandboxID string, svc runtimeAPI.TTRPCSandboxService) {
+	// Let the shim exit, then we can clean up the bundle after.
+	if _, sErr := svc.ShutdownSandbox(ctx, &runtimeAPI.ShutdownSandboxRequest{
+		SandboxID: sandboxID,
+	}); sErr != nil {
+		log.G(ctx).WithError(sErr).WithField("sandboxID", sandboxID).
+			Error("failed to shutdown sandbox")
+	}
+
+	ctx, cancel := context.WithTimeout(ctx, 5*time.Second)
```

Comment:
@fuweid I can open an issue to make this configurable as I know you had that feedback in the last change, although perhaps if we explore @dmcgowan's idea about trying to wire this up to garbage collection the solution will be more apparent.

Reply:
GC sounds good to me!